bebraven/platform
# Fix launch timezone

### Description

When launching a new program, use the timezone specified in Salesforce.



Task https://app.asana.com/0/1174274412967132/1198935151567000/f

Testing: 

1. Have a program in salesforce, with the timezone set (to something other than Eastern)
2. Launch a new program from platform, using the program ID from above
3. Check the settings of the new courses in Canvas, verify the timezone matches the salesforce program's timezone

### Motivation and Context




### Types of changes




### Checklist:
